responses/create-azure-resources.md

```diff
@@ -103,12 +102,12 @@ Let's take a quick peek at the jobs in this workflow before it get's triggered a
      - name: Configure probot app secrets
        if: success()
        run: |
-         az webapp config appsettings set  --name ${{ env.AZURE_WEBAPP_NAME }} --resource-group ${{ env.AZURE_RESOURCE_GROUP }} --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}} --settings APP_ID=${{secrets.PROBOT_APP_ID}} PRIVATE_KEY=${{secrets.PROBOT_APP_PRI_KEY}} WEBHOOK_SECRET=${{secrets.PROBOT_WEBHOOK_SECRET}}
+         az webapp config appsettings set  --name ${{ env.AZURE_WEBAPP_NAME }} --resource-group ${{ env.AZURE_RESOURCE_GROUP }} --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}} --settings APP_ID=${{secrets.PROBOT_APP_ID}} 
```

Comment:
@davenowell only `PRIVATE_KEY` needs to be removed.  The `WEBHOOK_SECRET` copies perfectly fine.